layouts/shortcodes/adoptions.html

```diff
@@ -0,0 +1,19 @@
+{{ $key       := .Get 0 }}
+{{ $adoptions := index site.Data.adoptions $key }}
+<br />
+
+<div class="">
+  <div class="adoptions-container">
+    {{ range $adoptions }}
+    {{ $img    := .tag | lower }}
+    {{ $type   := .img_type | default "png" }}
+    {{ $alt    := .alt | default .tag }}
+    {{ $imgUrl := printf "community/adoptions/img/logos/%s_logo.%s" $img $type | relURL }}
```

Comment:
Note that the compound file names use hyphens not underscores. Also, we can use a relative path, as I'd mentioned in the previous round of edits:

```suggestion
    {{ $imgUrl := printf "img/%s-logo.%s" $img $type }}
```